components/availability/src/Availability.svelte

```diff
@@ -39,11 +44,20 @@
 
   //#region mount
   let manifest: Partial<Manifest> = {};
+  $: calendarID = "";
   onMount(async () => {
     await tick();
     clientHeight = main?.getBoundingClientRect().height;
     const storeKey = JSON.stringify({ component_id: id, access_token });
     manifest = (await $ManifestStore[storeKey]) || {};
+    const calendarQuery: CalendarQuery = {
+      access_token,
+      component_id: id,
+      calendarIDs: [], // empty array will fetch all calendars
+    };
+    const calendarsList = await CalendarStore.getCalendars(calendarQuery);
+    calendarID = calendarsList.find((cal) => cal.is_primary)?.id || "";
+    console.log(calendarID);
```

Comment:
✂️

Reply:
This was intended. It will be removed when we add POST event for creating events